src/LogoFX.Client.Core.Platform/src/CommonProperties.cs

```diff
@@ -14,6 +20,7 @@ public class CommonProperties
         /// </summary>
         /// <param name="obj"></param>
         /// <returns></returns>
+        [SuppressMessage("ReSharper", "UnusedMember.Global")]
```

Comment:
@glyad Please remove - no need to add dependency on Resharper here

Reply:
I have not added dependency on Resharper.  It's like an annotation. Need to discuss it really.

Reply:
In conclusion to our discussion, I decided to remain it